settings.gradle.kts

```diff
@@ -1,13 +1,19 @@
 rootProject.name = "junit-pioneer"
-include("demo")
 
 plugins {
-	id("com.gradle.enterprise") version "3.14.1"
+	id("com.gradle.enterprise") version "3.16.2"
 }
 
 gradleEnterprise { 
 	buildScan {
 		termsOfServiceUrl = "https://gradle.com/terms-of-service"
 		termsOfServiceAgree = "yes"
 	}
-}
\ No newline at end of file
+}
+
+// See https://community.sonarsource.com/t/sonar-gradle-skipcompile-is-not-working/102710/9
+// TODO move this to a better place once https://sonarsource.atlassian.net/browse/SONARGRADL-134 is resolved.
+// > The 'sonar' / 'sonarqube' task depends on compile tasks.
+// > This behavior is now deprecated and will be removed in version 5.x.
+// > To avoid implicit compilation, set property 'sonar.gradle.skipCompile' to 'true' and make sure your project is compiled, before analysis has started.
```

Comment:
This needs to be a proper issue if/when we merge this, I think.

Reply:
Do you mean create an issue for the TODO to be fixed? (it'll linger for a long time, as there's no progress on it)

The warning (`>` text) is solved, the CI is doing a full build before running sonar:

https://github.com/junit-pioneer/junit-pioneer/blob/b02cda658464d548c011a1efb651cbf875a2dea4/.github/workflows/build.yml#L95-L103

Reply:
@Michael1993 can you please clarify or resolve?

Reply:
Yes, I meant the `TODO` comment should be a proper issue. My feeling on `TODO` in code is that it is more likely to be forgotten than a proper issue (even though those also tend to get forgotten after a bit...)

Reply:
Since this Todo is based on external events which may take months or never get done. I think it's enough for the next person who touches that area, as a reminder to open the issue and see what's up.